### ISSUE TITLE
Wx GUI Zoom Error

### ISSUE THREAD

**Issue by ocsmit** [HuidaeCho/projpicker#48]
Getting this error when trying to Zoom.

```
Traceback (most recent call last):
  File "/home/ocsmit/Work/projpicker/projpicker/gui_wx.py", line 255, in on_zoom
    geoms_bbox = calc_geoms_bbox()
TypeError: calc_geoms_bbox() missing 1 required positional argument: 'geoms'
```

**Comment by HuidaeCho** [comment]
Fixed in 82ef7d6

**Comment by HuidaeCho** [comment]
@ocsmit Please test the wx GUI extensively. I may have left some tk code there.
